.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
```

Comment:
I don't think we should prepopulate answers in my opinion.

Especially as on Linux, it's not often that distributions will come with the latest version of any particular tool, so `latest` isn't a very good assumption/default either.

If we were to prepopulate it, I'd go for `unspecified`, but even then, I think leaving it blank is best.

Reply:
Blank is definitely best, everything else is left for the user to fill, why make this one different